### PR TITLE
Updated the release process for Symfony 3.x and future releases

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: contributing/community/releases.rst [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
-.. tip::
-
-    The meaning of "minor" comes from the `Semantic Versioning`_ strategy.
-
-Each minor version sticks to the same very well-defined process where we start
-with a development period, followed by a maintenance period.
+* A new Symfony minor version (e.g. 2.8, 3.2, 4.1) comes out every *six months*:
+  one in *May* and one in *November*;
+* A new Symfony major version (e.g., 3.0, 4.0) comes out every *two years* and
+  it's released at the same time of the last minor version of the previous branch.
```
Comment (xabbuh) [inline contributing/community/releases.rst]: [...] the previous major version.

File: contributing/community/releases.rst [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
 
 .. note::
 
-    This release process has been adopted as of Symfony 2.2, and all the
-    "rules" explained in this document must be strictly followed as of Symfony
-    2.4.
+    This release process has been adopted as of Symfony 2.2, and all the "rules"
+    explained in this document must be strictly followed as of Symfony 2.4.
```
Comment (xabbuh) [inline contributing/community/releases.rst]: Well, the two year release cycle for major versions was not adopted in Symfony 2.2, but started with Symfony 3.0.

File: contributing/community/releases.rst [hunk 3]
```diff
@@ -74,6 +79,10 @@ fixes, and for a *four year* period for security issue fixes.
     Paid support after the three year support provided by the community can
     also be bought from `SensioLabs`_.
 
+In Symfony 2.x branch, the LTS versions are 2.3, 2.7 and 2.8. Starting from 3.x
+branch, only the last minor version of each branch is considered LTS (e.g 3.4,
```
Comment (xabbuh) [inline contributing/community/releases.rst]: missing dot after "e.g"

File: contributing/community/releases.rst (hunk 2 above)
Comment (javiereguiluz) [inline contributing/community/releases.rst]: I propose to remove this note entirely. It made sense when we did the transition to the new formal process. But that was years ago!

Comment (xabbuh) [inline contributing/community/releases.rst]: makes sense too (people are probably not so interested in how the process was in the past but how/when releases will be done in the future)

**Comment by rvanlaak** [comment]
Shouldn't the development phase in the image always start at the release date of the previous milestone? The 3.1 version does not seem to match that. Or won't be there any development in the next couple of months? :+1:

**Comment by javiereguiluz** [comment]
@rvanlaak you are right! The image contained lots of errors. Hopefully all are gone in 99070c9. Thanks for this nice catch and for warning me about this.

**Comment by weaverryan** [comment]
Thanks Javier! It reads really nicely :)

**Pull request merged**